packages/pn-pa-webapp/src/models/Statistics.ts

```diff
@@ -0,0 +1,162 @@
+export enum NotificationStatus {
+  ACCEPTED = 'ACCEPTED',
+  DELIVERING = 'DELIVERING',
+  DELIVERED = 'DELIVERED',
+  VIEWED = 'VIEWED',
+  EFFECTIVE_DATE = 'EFFECTIVE_DATE',
+  UNREACHABLE = 'UNREACHABLE',
+  CANCELLED = 'CANCELLED',
+  REFUSED = 'REFUSED',
+}
+
+export enum DeliveryMode { // export from pn-commons/../NotificationDetail.ts???
```

Comment:
@maurizioflauti to address this comment: no, they should not be exported from NotificationDetail types

Reply:
ok